client/contivModelClient.go

```diff
@@ -492,8 +501,23 @@ type TenantLinkSets struct {
 	Volumes        map[string]Link `json:"Volumes,omitempty"`
 }
 
+type TenantOper struct {
+	Endpoints        []EndpointOper `json:"endpoints,omitempty"`
```

Comment:
Since NetworkOper below will have the endpoints anyway, duplicating it here is not very useful i think..